docs/src/DeveloperDocumentation/printing_details.md

```diff
@@ -248,7 +248,7 @@ The `IOCustom` object allows one to locally control:
 
 We illustrate this with an example
 
-```
+```julia
```

Comment:
this example (together with the show demonstration below) could also be converted into a doctest